### PR TITLE
search: enables fc keyword

### PR DESCRIPTION
**Pull request by Panos512**
* Need to be merged with inspirehep/invenio-query-parser#15 .

Signed-off-by: Panos Paparrigopoulos <panos.paparrigopoulos@cern.ch>

### CODE REVIEW

File: tests/unit/search/search_test_queries.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
```
Comment (jacquerie) [inline tests/unit/search/search_test_queries.py]: Filename should be: `tests/unit/search/test_search_test_query.py

File: tests/unit/search/search_test_queries.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+def test_fc_keyword():
+    """
+    Test queries using field_code keyword (fc) .
+    """
+    spires_query = InspireQuery('find fc a')
+    invenio_query = InspireQuery('fc: a')
+
+    assert (spires_query.body == {'query': {'multi_match': {'query': 'a', 'fields': ['field_code']}}})
+    assert (invenio_query.body == {'query': {'multi_match': {'query': 'a', 'fields': ['field_code']}}})
```
Comment (jacquerie) [inline tests/unit/search/search_test_queries.py]: Parentheses are not needed around `assert`s, so you can remove them.

File: tests/unit/search/search_test_queries.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+# License, or (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
```
Comment (jacquerie) [inline tests/unit/search/search_test_queries.py]: Please add CERN paragraph.

**Comment by Panos512** [comment]
This is failing because of  inspirehep/invenio-query-parser#15 (uses wrong keyword-mapping) .

**Comment by Panos512** [comment]
This is failing because of inspirehep/invenio-query-parser#15 (uses wrong keyword-mapping) .

File: tests/unit/search/test_search_queries.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+def test_fc_keyword():
+    """
+    Test queries using field_code keyword (fc) .
+    """
+    query_using_spires_syntax = InspireQuery('find fc a')
+    query_using_invenio_syntax = InspireQuery('fc: a')
+
+    assert query_using_spires_syntax.body == {'query': {'multi_match': {'query': 'a', 'fields': ['field_code']}}}
+    assert query_using_invenio_syntax.body == {'query': {'multi_match': {'query': 'a', 'fields': ['field_code']}}}
```
Comment (jacquerie) [inline tests/unit/search/test_search_queries.py]: This approach is kind-of-superseded by the one in #1016, which will be merged soon™. Only the second test will survive as `test_field_code_colon` and with a structure similar to the other tests in `tests/unit/search/test_search_query.py`.

**Comment by jacquerie** [comment]
After merging #976 this needs a rebase. Sorry!

**Pull request merged**